configmap/repl-wrk/start/start.sh

```diff
@@ -39,10 +40,10 @@ fi
 
 HOST_DN=$(hostname --fqdn)
 
-# Wait for repl-db started
-# and contactable
+# Wait for remote repl-db started and contactable
```

Comment:
It's not needed because unlike Qserv all components of the Replication system would be making repeated attempts to connect to the database service (and to each other). This allows staring the components in an arbitrary order w/o waiting.
The only exception (in the current implementation of the system) is a host where the`xrootd` runs. If this host is not known in DNS (by name or IP address) then `Master Replication Controller` will fail. This problem will be fixed in a future implementation of the system.

Reply:
ok, `repl-wrk` stands for `replication worker` and run int the same pod that xrootd server. I understand this on/off DNS problem has appeared with k8s, be aware that DNS entry for a worker can disappear and appear again each time a service fails inside the worker pod. Please let me know when it is fixed.